services/mirror/mirror_pull.go

```diff
@@ -0,0 +1,410 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
2021

Reply:
Even if the code in the file was just moved from the other file?